app/assets/javascripts/cable.js

```diff
@@ -5,9 +5,8 @@
 //= require_self
 //= require_tree ./channels
 
-(function() {
-  this.App || (this.App = {});
+(function () {
+  this.App || (this.App = {})
 
-  App.cable = ActionCable.createConsumer();
-
-}).call(this);
+  App.cable = ActionCable.createConsumer() //eslint-disable-line
```

Comment:
My question is: if this is not auto-generated, why are we disabling listing for this line of code?

Reply:
this isn't auto generated. I manually put it in, because eslint complains about `ActionCable` and `App` not being defined. Stimulus, afaict doesn't have an eslint rules package, so we will likely need to occasionally bypass lint rules in this manner.